src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  width: 100% - 100px;
+}
+
+.logo_img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: inline-block;
+  text-align: center;
+
+}
+
+.nav__link {
+  text-decoration: none;
+  display: inline-block;
+  color: black;
+  line-height: 60px;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  display: block;
+  content: "";
+  position: absolute;
+  background-color: #00acdc;
+  height: 4px;
+  width: 37px;
+  border-radius: 2px;
+  top: 56px;
+}
+
+.nav__link__margin {
+  margin: 0 10px;
+}
+
+.nav__link__margin__apple {
+  margin-right: 10px;
+}
```

Comment:
Instead of this use [:not(:last-child)](https://medium.com/design-code-repository/css-selector-not-last-child-48b5192f0b50) for main class. And remove this class from html